lib/tasks/ci.rake

```diff
@@ -223,7 +223,12 @@ namespace :dcv do
       yml_file = File.join(Rails.root, 'config/location_uris.yml')
       FileUtils.touch(yml_file) # Create if it doesn't exist
       stub_yml = YAML.load_file(yml_file) || {
-        '_all_environments' => {
+        'development' => {
```

Comment:
This comment isn't related to your changes here, but just a note about rake tasks that generate yml files.  At some point we may want to consider creating a /config/templates directory and copy template yml files from there to /config instead.  I think the files are much easier to view and edit that way.

As an example, I just added it to amesa in this PR: https://github.com/cul/ldpd-amesa/pull/4/files#diff-c750a294b07117e50e2c9487023db2494ae143a77588ce504886849ca4fea141R3-R22

Reply:
There actually is a `config/templates` 😆 this was just an incremental change.

Reply:
Oh!  In that case, I'm now even more in favor of moving this stuff over there, haha.

Reply:
Ticketing this as a patch/maintenance task https://issues.cul.columbia.edu/browse/DLC-876